src/os/lsof_utils.rs

```diff
@@ -62,7 +62,7 @@ impl RawConnection {
 }
 
 pub fn get_connections<'a>() -> RawConnections {
-    let content = run(&["-n", "-P", "-i4"]);
+    let content = run(&["-n", "-P", "-i4", "+c", "30"]);
```

Comment:
When I do `lsof -n -P -i4 +c 30` on my machine, I get `lsof: +c 30 > what system provides (15)`
I looked through [the lsof man page](https://linux.die.net/man/8/lsof) and under the `+c w` section, I found this: "If w is zero ('0'), all command characters supplied to lsof by the UNIX dialect will be printed"

What do you think about changing "30" to "0"? That way we just get the max possible value on each machine?

Reply:
If our UI will correctly truncate the string to size of user's terminal, sure.

Reply:
Hmm looks like most processes on my machine have names < 15 chars, I don't think that will be a problem for many people. Let me update the code

Reply:
Also - we should truncate them in the UI. I think it already does that with the connection strings, which are horribly long. :)